### PR TITLE
Update cd-to to 2.7 of a maintained fork

### PR DESCRIPTION
**Pull request by tghs**
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/touche.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'touche' do
```
Comment (reitermarkus) [inline Casks/touche.rb]: Please create a separate PR for this.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by tghs** [comment]
Removed the other commit accidentally included in this PR.

**Comment by vitorgalvao** [comment]
This needs to be renamed to prepend `ealeksandrov` to the name (as we do with other forks).

It does not matter if the original was abandoned and this one is maintained. It matters that users will be expecting the original and not *a* (non-specific) fork.

Exceptions to this rule (i.e. a fork becomes the main one without needing a prefix) occur when the original repo links to a fork as the new maintained version (i.e. the author is officially endorsing a fork as the new original).

**Comment by tghs** [comment]
@vitorgalvao Cheers for the explanation. I'll close this and submit a fresh PR with it renamed.

**Comment by vitorgalvao** [comment]
@tghs It’s fine if you just edit this one. We can squash everything when accepting.

**Pull request closed without merging**